source/declarations.tex

```diff
@@ -6900,12 +6900,11 @@
 An \grammarterm{enumerator-definition} with \tcode{=} gives the associated
 \grammarterm{enumerator} the value indicated by the
 \grammarterm{constant-expression}.
-If the first \grammarterm{enumerator}
-has no \grammarterm{initializer}, the value of the corresponding constant
-is zero. An \grammarterm{enumerator-definition} without an
-\grammarterm{initializer} gives the \grammarterm{enumerator} the value
-obtained by increasing the value of the previous \grammarterm{enumerator}
-by one.
+An \grammarterm{enumerator-definition} without \tcode{=} gives the associated
+\grammarterm{enumerator} the value obtained by
+increasing the value of the previous \grammarterm{enumerator} by one.
+If the first \grammarterm{enumerator-definition} has no \tcode{=}, the associated
+\grammarterm{enumerator} is given the value zero.
```

Comment:
The last sentence doesn't sound like an exception to the first. It also reads better when the zero case is upfront.
```suggestion
\grammarterm{enumerator} the value zero
if it is the first \grammarterm{enumerator-definition},
and the value of the previous \grammarterm{enumerator}
increased by one otherwise.
```

Reply:
FWIW, I like that suggestion.

Reply:
@Quuxplusone Let's apply it!